### ISSUE TITLE
Enter radix is not allowed with a mask number and allowNegative set to "false"

### ISSUE THREAD

**Issue by jubaron** [uNmAnNeR/imaskjs#156]
Enter radix is not allowed with a mask number and allowNegative set to "false"
This issue is due to "midInput" variable: midInput += '(0|([1-9]+\\d*))'; I change to  midInput += '(['+escapeRegExp(this.radix)+']?|(0|([1-9]+\\d*)))';

**Comment by uNmAnNeR** [comment]
@jubaron 
just checked when `allowNegative = false`:
```js
> start + midInput + end
"^(0|([1-9]+\d*))(,\d{0,2})?$"
> start + mid + end
"^\d*(,\d{0,2})?$"
```
Both regex work with radix. I don't see any error in the code for now, please provide more details. At least check both compiled regex. May be some other options are invalid.

**Comment by jubaron** [comment]
The problem in my case is: if I enter.25 for example, in order to have automatically 0.25 it's not possible. I need to start with a number. With allowNegative set to true it's possible.

**Comment by uNmAnNeR** [comment]
@jubaron ok, finally i reproduced it. Well, actually i supposed that it not possible to start input from radix :) So probably this is a bug when `allowNegative` is true, not false. But ok, for now i will try to find solution to allow radix in the beginning for both cases. Thanks for reporting.

**Comment by jubaron** [comment]
I my case and I think it's the best solution I would like to enter the radix first in order to autocomplete with the zero like the windows calculator. To do this I change the regex to: '(['+escapeRegExp(this.radix)+']?|(0|([1-9]+\d*)))'. It's work.

**Comment by uNmAnNeR** [comment]
@jubaron i can say this is the best solution, because `radix` is already included in the `end` part of regex.
So please try this one: https://github.com/uNmAnNeR/imaskjs/pull/154/files#diff-5dd76b379a802e072fd8cc1608d56766

**Comment by jubaron** [comment]
it works great. Do you plan to release new version with cdn ?

**Comment by uNmAnNeR** [comment]
@jubaron not now, i have several breaking changes and some features pending, so i plan to release major version soon. But soon might not become very soon :)

**Comment by uNmAnNeR** [comment]
fixed in #154
